test/cypress/support/output-logs.js

```diff
@@ -0,0 +1,50 @@
+// ***********
+// Output App logs in case a test fails!
+// Copied directly from: https://github.com/cypress-io/cypress/issues/3199#issuecomment-466593084
+// ***********
+
+let logs = ''
+
+Cypress.on('window:before:load', (window) => {
+  // Get your apps iframe by id.
+  const docIframe = window.parent.document.getElementById("Your App: 'group-income-simple'")
+
+  // Get the window object inside of the iframe
+  const appWindow = docIframe.contentWindow;
+
+  // This is where I overwrite all of the console methods.
+  ['log', 'info', 'error', 'warn', 'debug'].forEach((consoleProperty) => {
+    appWindow.console[consoleProperty] = function (...args) {
+      /*
+         * The args parameter will be all of the values passed as arguments to
+         * the console method. ( If your not using es6 then you can use `arguments`)
+         * Example:
+         *       If your app uses does `console.log('argument1', 'arument2');`
+         *       Then args will be `[ 'argument1', 'arument2' ]`
+         */
+      // Save everything passed into a variable or any other solution
+      // you make to keep track of the logs
+      // Use JSON.stringify to avoid [object, object] in the output
+      logs += JSON.stringify(args.join(' ')) + '\n'
```

Comment:
Hmm, can you please make sure to log the `consoleProperty` as a prefix (e.g. `[log]`, `[error]`, etc.)?

This way one can search to see what type of log is being outpout.

Reply:
Added!